sos/plugins/canonical_livepatch.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2016 Bryan Quigley <bryan.quigley@canonical.com>
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, UbuntuPlugin
+
+
+class CanonicaLivepatch(Plugin, UbuntuPlugin):
+    """Canonical Livepatch Service
+    """
+
+    plugin_name = 'canonical_livepatch'
+    profiles = ('system', 'kernel')
+    files = ('/snap/bin/canonical-livepatch')
+
+    def setup(self):
+        self.add_cmd_output(
+            "systemctl status snap.canonical-livepatch.canonical-livepatchd")
+        self.add_cmd_output("snap run canonical-livepatch status --verbose")
+        self.add_cmd_output("snap run canonical-livepatch --version")
```

Comment:
Why not use the list support in `add_cmd_output()`? Then these three lines can be written indented, and the funny line break in the 1st one (to meet the pep8 line length limit) disappears.

Anyway; not worth holding this up for - I'll go ahead and merge it now & then tweak it a bit.